ansible/MANUAL_STEPS.md

```diff
@@ -1,5 +1,17 @@
 # Manual steps required to run ansible on machines
 
+## RHEL7-S390X
+1. V8 build-tools
+
+Copy `/home/iojs/build-tools/{gn,ninja}` from any other RHEL7-S390X host that
+has them to the newly provisioned one.
```

Comment:
before copying across check the gn binary is valid by running `gn --version` which should return a version number and not `UNKNOWN`. Also I think the link for ninja is wrong.

Reply:
ok. current state needs fixing, I guess?
```
core/build (add-aix71-testing $% u=) % parallel-ssh -h hosts/rhel7-s390x -i "sudo /home/iojs/build-tools/gn --version" 
[1] 11:33:18 [FAILURE] test-ibm-rhel7-s390x-3 Exited with error code 1
Stderr: sudo: /home/iojs/build-tools/gn: command not found
[2] 11:33:18 [SUCCESS] test-ibm-rhel7-s390x-2
1620 (570aaed)
[3] 11:33:18 [SUCCESS] test-ibm-rhel7-s390x-4
UNKNOWN
[4] 11:33:19 [SUCCESS] test-ibm-rhel7-s390x-1
UNKNOWN
core/build (add-aix71-testing $% u=) % parallel-ssh -h hosts/rhel72-s390x -i "/home/iojs/build-tools/gn --version" 
[1] 11:33:32 [SUCCESS] test-linuxonecc-rhel72-s390x-2
UNKNOWN
[2] 11:33:32 [FAILURE] test-linuxonecc-rhel72-s390x-3 Exited with error code 127
Stderr: bash: /home/iojs/build-tools/gn: No such file or directory
[3] 11:33:33 [SUCCESS] test-linuxonecc-rhel72-s390x-1
1620 (570aaed)
```